### ISSUE TITLE
strict rule report smaller loc range

### ISSUE THREAD

**Issue by Aladdin-ADD** [eslint/eslint#10729]
**What rule do you want to change?**
`strict`
**Does this change cause the rule to produce more or fewer warnings?**
same
**How will the change be implemented? (New option, new default behavior, etc.)?**
n/w
**Please provide some example code that this change will affect:**

```js

```

**What does the rule currently do for this code?**
![image](https://user-images.githubusercontent.com/13050025/43674996-15fefd84-980f-11e8-8c3e-2611c26ab134.png)

**What will the rule do after it's changed?**
it make users hard to read and cannot see other errors-- I hope it not report the whole file.
